pullrequests.go

```diff
@@ -572,6 +591,31 @@ func (api *PullRequestsAPI) GetPullRequest(pullRequestNumber int) (*PullRequestR
 	return &pullRequest, nil
 }
 
+// MergePullRequest merges a pull request using the specified method.
+func (api *PullRequestsAPI) MergePullRequest(pullRequestNumber int, method MergeMethod) (*MergeRequestResponse, error) {
```

Comment:
We may want to add `sha` or other optional parameters at some point. `ghapi` is a `v0.x` project so we're free to break the API from time to time.

https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button